app.js

```diff
@@ -0,0 +1,8 @@
+function myFunction() {
+    var x = document.getElementById("myLinks");
+    if (x.style.display === "block") {
```

Comment:
Transfer this logic in css with class "active"
Then when you get the element with Js you can use `element.classList.toggle("active")`